### ISSUE TITLE
Micromamba 1.5.2 requires VSCpp Redistributables because of /MD

### ISSUE THREAD

**Issue by FelipeCalliari** [mamba-org/mamba#2928]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### How did you install Mamba?

Micromamba

### Search tried in issue tracker

it is a problem with the latest version of micromamba 1.5.2

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not applicable

### Describe your issue

The following "**enhancement**" (see below) for micromamba 1.5.2 forces users to install the latest "Visual Studio C++ Redistributables", but there is no advice in the README.

```
Release 1.5.2
Enhancement:
 - [libmamba] Build micromamba with **/MD** by @AntoinePrv in https://github.com/mamba-org/mamba/pull/2862
```

If someone tries to use micromamba 1.5.2 in a **fresh Windows installation**, they will be presented with error messages saying:

```
The code execution cannot proceed because VCRUNTIME140_1.dll was not found. Reinstalling the program may fix this problem.
```

And clearly it's not a micromamba problem, but rather the lack of VSCpp redistributables that can be downloaded through:

https://aka.ms/vs/17/release/vc_redist.x64.exe for x64
https://aka.ms/vs/17/release/vc_redist.x86.exe for x86

### mamba info / micromamba info

```shell
On a fresh Windows installation, the user can't run this command due to the lack of VSCpp redistributables!
```

### Logs

```shell
On a fresh Windows installation, the user can't run this command due to the lack of VSCpp redistributables!
```

**Comment by isuruf** [comment]
This also introduces a dependency for Miniforge when we switch to micromamba as the bootstrapping executable.

cc @AntoinePrv

**Comment by AntoinePrv** [comment]
@Klaim @JohanMabille

**Comment by JohanMabille** [comment]
I will amend the documentation, building with `/MT` means having all the dependencies built with `/MT`, and this prevents mixing static and dynamic linking. Or we need to provide 2 versions of the static builds of all the dependencies, one with `/MT` and one with `/MD`.

@isuruf is this dependency really problematic?

**Comment by FelipeCalliari** [comment]
@JohanMabille while @isuruf doesn't answer, I'll leave my opinion.

I don't think this dependency is problematic. Miniconda bundles in its installer the DLLs: msvcp140.dll, vccorlib140.dll, vcomp140.dll, vcruntime140.dll, vcruntime140_1.dll, etc.

So, if you release mamba/micromamba for Windows **with these DLLs**, it is fine. It will work out of the box on a fresh Windows installation.

**Comment by frankier** [comment]
This still seems to be a problem with the current micromamba. Perhaps this issue could be reopened?

**Comment by arthursw** [comment]
Micromamba is still not bundled with the DLLs ; so it requires the user to install the VSCpp redistributables manually, with administrative privileges. 

This is a real problem for apps relying on micromamba to install conda environments.

I am developping an application with a plugin system which automatically downloads micromamba to create and execute any conda environment. Thus, each plugin is isolated and this is totally transparent for the user.

It would be very useful to provide one micromamba version which bundles the required DLLs , or one that does not require the DLLs.

**Comment by arthursw** [comment]
The required DLLs are :

```
msvcp140.dll
vcruntime140.dll
vcruntime140_1.dll
```

**Comment by arthursw** [comment]
Another possibility is to add a command in the install script to silently install the redistributables. For example:

`Invoke-WebRequest -URI "https://aka.ms/vs/17/release/vc_redist.x64.exe" -OutFile "$env:Temp\vc_redist.x64.exe"; Start-Process "$env:Temp\vc_redist.x64.exe" -ArgumentList "/quiet /norestart" -Wait; Remove-Item "$env:Temp\vc_redist.x64.exe"`

**Comment by arthursw** [comment]
I agree, but this installation must be executed as admin ; so it's not ideal for an automatic installation.

How come `pixi` does not have this problem? 
Could the `pixi` solution be used for `micromamba`?

**Comment by arthursw** [comment]
Actually I was (very likely) wrong [this installation *does not* require administration rights](https://stackoverflow.com/questions/41011725/can-i-install-visual-studio-without-admin-rights).

**Comment by RoyiAvital** [comment]
> So it would be nice to add a command like the following in the install script to silently install the redistributables:

Have you tried it? I don't think it will install it at user level.

Anyhow, why not just compile with the static libraries and get rid of this dependency?
Alternatively, supply those DLL's in the ZIP file?
